.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,20 @@
+name: Dependency Graph
+on:
+  push:
+    branches:
+      - main
```

Comment:
We cannot activate it on pull requests because the GITHUB_TOKEN permission is `read` in pull requests